script/verify_security_report.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require File.expand_path('../config/environment', __dir__)
+require 'net/http'
+require 'uri'
+require 'logger'
+
+logger = Logger.new('log/verify_security_report.log')
+
+Validator.where.not(security_report_url: [nil, '']).find_each do |validator|
+  url = validator.security_report_url
+  begin
+    uri = URI.parse(url)
+    response = Net::HTTP.get_response(uri)
+    puts response.code.to_i
+    if response.code.to_i == 404
+      logger.info("REMOVED: #{validator.account} | #{url}")
+      validator.update(security_report_url: nil)
+    else
+      logger.info("OK for #{validator.account} (#{url})")
+    end
+  rescue SocketError, Errno::ECONNREFUSED, Errno::ETIMEDOUT, Net::OpenTimeout, Net::ReadTimeout, URI::InvalidURIError => e
```

Comment:
I don't think we should remove url right away if the website is unreachable - there might be some temporary problems (I'm refering to timeouts, not eg. InvalidURIError - in that case I think it's OK to remove).